google/cloud/forseti/services/cli.py

```diff
@@ -203,13 +203,108 @@ def define_inventory_parser(parent):
         help='Inventory id to get')
 
 
+def define_config_parser(parent):
+    """Define the config service parser.
+
+    Args:
+        parent (argparser): Parent parser to hook into.
+    """
+
+    service_parser = parent.add_parser('config', help='config service')
```

Comment:
The config will support choosing endpoint, key and model handler. Is there anything else? Do you think it makes sense to be more comprehensive on the help of the config?